.github/workflows/ci.yaml

```diff
@@ -49,24 +49,27 @@ jobs:
       contents: read
       packages: write
 
-  supergraph_composition:
+  supergraph_update:
     # Deduplicate jobs from pull requests and branch pushes within the same repo.
     if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.repository
     needs: graph_proxy_schema
-    uses: ./.github/workflows/_supergraph_composition.yaml
-
-  supergraph_update:
-    # Only update the supergraph on releases (tagged pushes to main)
-    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
-    needs: supergraph_composition
-    uses: ./.github/workflows/_supergraph_update.yaml
-    secrets:
-      GRAPH_FEDERATOR: ${{ secrets.GRAPH_FEDERATOR }}
-    
+    runs-on: ubuntu-latest
+    steps:
+      - name: Update Supergraph
+        uses: diamondlightsource/graph-federation@main
```

Comment:
When do we recommend using `v1` vs `main`?

Reply:
We should probably actually make them all `main` right now as we've not yet cut a release

Reply:
Have changed them all to `main`for now